### PR TITLE
[Bug]fix bug in handle multiple quotes

### PR DESCRIPTION
**Pull request by eason-lyx**
when handle data with quotes,we need to follow the rules:
1)the cell should be enclosed in quote character
2)inside the data eache quote is escaped using escape character.

for example:
"""abc""" -> "abc" (escape char and quote is ")
”\"i\"" ->"i" (escape char is ,quote is ")

### CODE REVIEW

File: integration/spark/src/test/scala/org/carbondata/spark/testsuite/dataload/TestLoadDataWithHiveSyntax.scala [hunk 1]
```diff
@@ -273,6 +274,23 @@ class TestLoadDataWithHiveSyntax extends QueryTest with BeforeAndAfterAll {
...
+
+    sql("""
+        CREATE table t3(customer_id int, 124_string_level_province String, date_level String,
+        Time_level String, lname String, fname String, mi String, address1 String, address2 String, address3 String, address4 String, city String, country String, phone1 String, phone2 String, marital_status String, yearly_income String, gender String, education String, member_card String, occupation String, houseowner String, fullname String, numeric_level double, account_num double, customer_region_id int, total_children int, num_children_at_home int, num_cars_owned int)
+        STORED BY 'org.apache.carbondata.format'
+        """)
+
+    sql("""
+       LOAD DATA LOCAL INPATH './src/test/resources/datawithcomplexspecialchar.csv' into
+       table t3 options ('DELIMITER'=',', 'QUOTECHAR'='"','ESCAPECHAR'='"')
```
Comment (Vimal-Das) [inline integration/spark/src/test/scala/org/carbondata/spark/testsuite/dataload/TestLoadDataWithHiveSyntax.scala]: can you add one more test with " as quote character and \ as escape character
Data should be "\"i\""

**Comment by eason-lyx** [comment]
@Vimal-Das comment is handled.
please check.

**Comment by Vimal-Das** [comment]
retest this please

**Comment by Vimal-Das** [comment]
LGTM, Merged to master.

**Pull request merged**